nss-day-cohort-46/holiday-road-faint-dusk
# Mp weather provider

# Description

Built out the weather forecast modules and added initial styling to weather cards

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Please check that the weather shows up in the DOM and there are no errors in the console.

```
git fetch --all
git checkout mp-weather-prov
```

# Checklist:
